bin/n

```diff
@@ -1195,6 +1222,60 @@ function get_engine_version() {
   [[ -n "${g_target_node}" ]] || abort "did not find supported version of node in 'engines' field of package.json"
 }
 
+#
+# Synopsis: version_satisfies_target current target
+# Check if current version satisfies the given target version or range.
+#
+
+function version_satisfies_target() {
+  local current="$1"
+  local target="$2"
+  
+  # Remove 'v' prefix if present
+  current="${current#v}"
+  target="${target#v}"
+  
+  # Exact match or wildcard
+  if [[ "$current" == "$target" || $target == "*" ]]; then
+    return 0
+  fi
+  
+  # Check for operator-based range (only supports what get_package_engine_version does)
```

Comment:
This code closely follows get_package_engine_version which only has to deal with things that are in engines fields. You might be assuming that everything is a range by now, but `latest` and `current` are currently coming through as strings. This triggers the semver call which does not recognise the label.